README.md

```diff
@@ -219,6 +219,40 @@ API along with your API token.
 }
 ```
 
+- Specify a url to use for GTFS-RT updates:
+
+```json
+{
+  "agencies": [
+    {
+      "url": "http://countyconnection.com/GTFS/google_transit.zip",
+      "rtupdates": [
+        "https://opendata.somewhere.com/gtfs-rt/updates.pb"
+      ],
+    }
+  ]
+}
+```
+
+- Specify serveral urls to use for GTFS-RT updates and custom headers:
+
+```json
+{
+  "agencies": [
+    {
+      "url": "http://countyconnection.com/GTFS/google_transit.zip",
+      "rtupdates": [
```

Comment:
The key `rtupdates` might be confusing, and it might be best to require users to specify which type of GTFS-RT each URL is.  Here is a proposed config:

```json
 {
   "agencies": [
     {
       "url": "http://countyconnection.com/GTFS/google_transit.zip",
       "gtfs_rt_urls": {
         "realtime_alerts": "https://opendata.somewhere.com/gtfs-rt/Alerts.pb",
         "realtime_trip_updates": "https://opendata.somewhere.com/gtfs-rt/TripUpdates.pb",
         "realtime_vehicle_positions": "https://opendata.somewhere.com/gtfs-rt/VehiclePositions.pb",
       },
       "gtfs_rt_headers": {
         "Authorization": "bearer 1234567890"
       }
     }
   ]
 }
 ```

Reply:
I was doing this initially but then did the array as I started to do it more "agile" for any future files etc, but perhaps it should be separated again for clarity? I am thinking that I will continue to do a lot of additional changes such as adding types and tests which are not yet done. Looking forward to more feedback =)